apps/dashboard/app/views/files/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 
-<pre id="editor" data-path="<%= @path %>" data-api="<%= file_api_url(@filesystem, @path) %>"><%= @content %></pre>
+<pre id="editor" data-path="<%= @path %>" data-api="<%= OodAppkit.files.url(path: @path, fs: @filesystem) %>"><%= @content %></pre>
```

Comment:
I'm wondering after #2240 if `OodAppkit.files.url` or `OodAppkit.files.api` is the correct one here.
It has previously been `files.api`, but the workaround in `files_helper` caused `OodAppkit.files.api` to not really be the actual API URL.
Though, both are exactly the same assuming the routes work as intended.

Reply:
See I knew I thought the files app used the api for a reason! It's because we did.

I could sort of go either way, but am leaning towards using the API almost solely because that's what it used to do. The comments in the routes say to deprecate the API, but that's not likely given we have no replacement. I don't believe it'll ever be a first class citizen with documentation and strong(er) stability, but it will be there.

So I guess I have the weak opinion to use the API just because that's what we've historically done (mostly because it used to be an entirely separate process that you could _only_ use an API to interact with).  So it has that one advantage of being in my memory.

Reply:
Changed this to use the API. This also has the benefit of adding some tests for #2242.